### ISSUE TITLE
Safari - Some thumbnails not rendering

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#569]
https://geo.btaa.org/?f%5Bdct_isPartOf_sm%5D%5B%5D=12d-04&per_page=10

### Safari
![Screenshot 2024-01-12 at 10 15 21 AM](https://github.com/geobtaa/geoportal/assets/69827/8b5bd481-4d27-4b86-87cb-de8c1c0a0341)

### Chrome / Firefox
![Screenshot 2024-01-12 at 10 16 09 AM](https://github.com/geobtaa/geoportal/assets/69827/a2af5f47-f608-49f8-8e05-83d47f2cd3e0)

**Comment by ewlarson** [comment]
More info here. Appears Safari is rejecting the original image too:
https://isac-idb-static.uchicago.edu/multimedia/3001247/1014604_APCS_Arabian_Sea_AP-20_3rd_Edition_1951_preview.1920x1200.jpg 

That URL will display an image in Chrome, but in Safari it will be blank.

**Comment by mberkowski** [comment]
@ewlarson Safari rejects this image outright from its original URL, or is it rejected if called via XHR or dynamically loaded?

**Comment by ewlarson** [comment]
Rejecting is the wrong word here... I don't know quite how to phrase it. Safari is receiving (response) and loading (there are bytes here) a JPEG it just does not render — preview is just an empty/blank canvas.

![Screenshot 2024-01-12 at 1 32 55 PM](https://github.com/geobtaa/geoportal/assets/69827/0277369c-60a5-4c92-89c7-6fcb98e23274)

**Comment by ewlarson** [comment]
Our thumbnail image in the Geoportal is really just a cropped variant of that original from U of Chicago... I don't know that there is anything here for us to resolve. We should pass this issue back to U of Chicago for their tech staff to troubleshoot.

**Comment by ewlarson** [comment]
One other thing! @njvack suggested if we run these JPEGs thru an image optimization tool (like [Kraken](https://kraken.io/)), it might just fix them and that does indeed seem to fix the example from above.

### Original (Does not load in Safari)
https://isac-idb-static.uchicago.edu/multimedia/3001247/1014604_APCS_Arabian_Sea_AP-20_3rd_Edition_1951_preview.1920x1200.jpg

### Kraken'd (Does load in Safari)
https://dl.kraken.io/web/80de08dfcda5c0dd9a04278115f0e3ae/1014604_APCS_Arabian_Sea_AP-20_3rd_Edition_1951_preview.1920x1200.jpg

**Comment by njvack** [comment]
Yeah, jpeginfo suggests that the originals are Fancy Jpegs. It's super weird that Safari won't render them — they show up fine with quicklook and Preview and hey I would have expected those tools to use the same image libraries as Safari but ¯\_(ツ)_/¯
